examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -391,6 +430,11 @@ def compute_metrics(eval_pred):
             checkpoint = last_checkpoint
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
         trainer.save_model()
+        for metric_dict in trainer.state.log_history:
+            if 'loss' in metric_dict:
```

Comment:
It seems to call mlflow API after training is done. 
We should not assume a pytorch process is normally done. 
Any metrics won't be logged in the case that the process is terminated.

In this view, my suggestion is to log metrics in `TBTrainerCallback `.  Do you have any idea?

Reply:
As I mention before, trainer log these metrics each log interval in trainer state. So those line of code just get the logged values in model states and save to mlflow. This way, we get the results as list with each metrics value associate with one log interval. Using `TBTrainerCallback `  to log these metrics every interval is trickier b/c we can get the result as list.

Reply:
As I known, mlflow.log_metric take a value or a list for a key word. I don't know if it can append the new value to the same key word each interval.